src/config/2019.json

```diff
@@ -746,6 +746,16 @@
       "avatar_url": "https:\/\/avatars2.githubusercontent.com\/u\/24447527?v=4&s=200",
       "github": "raghuramakrishnan71"
     },
+    "raghav": {
+      "name": "Raghvendra Kumar",
+      "bio":"Tech Enthusiast and Developer",
```

Comment:
At the moment only Author's bios are used (see #350) so other contributors don't get a bio - sorry! 😊

```suggestion
```